### PR TITLE
Tl 24: Implement Navbar

### PR DESCRIPTION
**Pull request by peterevilla**
[Jira story](https://triplist.atlassian.net/secure/RapidBoard.jspa?rapidView=1&modal=detail&selectedIssue=TL-24)

### What does this PR do?
Nav bar implemented

### Screenshots
B914D753-4F32-477F-931C-8EB012607FC4_4_5005_c

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (LuisRevillaM) [inline package-lock.json:1]: Let's delete this file.

File: package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "react-scripts": "3.4.1",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "yarn": "^1.22.4"
```
Comment (LuisRevillaM) [inline package.json:21]: Let's delete this line: `"yarn": "^1.22.4"`

File: yarn.lock [hunk 3]
```diff
@@ -11328,3 +11328,8 @@ yargs@^15.0.2:
     which-module "^2.0.0"
     y18n "^4.0.0"
     yargs-parser "^18.1.1"
+
+yarn@^1.22.4:
+  version "1.22.4"
+  resolved "https://registry.yarnpkg.com/yarn/-/yarn-1.22.4.tgz#01c1197ca5b27f21edc8bc472cd4c8ce0e5a470e"
+  integrity sha512-oYM7hi/lIWm9bCoDMEWgffW8aiNZXCWeZ1/tGy0DWrN6vmzjCXIKu2Y21o8DYVBUtiktwKcNoxyGl/2iKLUNGA==
```
Comment (LuisRevillaM) [inline yarn.lock:11335]: Let's delete this.

You can copy the `yarn.lock` from master and paste it on your branch's.

File: package-lock.json (hunk 1 above)
Comment (peterevilla) [reply package-lock.json]: done

File: package.json (hunk 2 above)
Comment (peterevilla) [reply package.json]: done

File: yarn.lock (hunk 3 above)
Comment (peterevilla) [reply yarn.lock]: done

**Pull request merged**